projects/admin/src/app/routes/route.service.ts

```diff
@@ -71,7 +72,7 @@ export class RouteService {
   initializeRoutes() {
     this._routeCollectionService
       .addRoute(new CirculationPoliciesRoute(this._routeToolService))
-      .addRoute(new DocumentsRoute(this._routeToolService))
+      .addRoute(new DocumentsRoute(this._routeToolService, this._route))
```

Comment:
Removing this argument (see previous message), all changes from this file could be remove.
`git checkout rero/stating -- projects/admin/src/app/routes/route.service.ts`